spec/runa/models/remote_code_spec.rb

```diff
@@ -18,32 +18,6 @@
       end
     end
 
-    context 'when URL is valid but not known' do
```

Comment:
I would re-instate a version of this test just to codify the expected behaviour. What happens? I don't know any more.